int32 speed_x
int32 speed_z
int32 power
int32 limit_down
int32 limit_up
int32 goal_point
int32 robot_start